torch/distributed/nn/api/remote_module.py

```diff
@@ -143,6 +151,19 @@ def __init__(
             method = torch.jit.export(method)
             setattr(self, method_name, types.MethodType(method, self))
 
+    def remote_parameters(self, recurse: bool = True) -> List[rpc.RRef[Parameter]]:
+        r"""Returns a list of RRefs of remote module parameters.
+        This is typically passed to a distributed optimizer.
+        Args:
+            recurse (bool): if True, then yields parameters of the remote module
```

Comment:
nit: maybe use `returns` instead of `yields` to avoid any confusion that we're using a generator here.